static/src/javascripts/bootstraps/enhanced.js

```diff
@@ -9,7 +9,9 @@ define([
     'common/utils/mediator',
     'common/utils/user-timing',
     'bootstraps/common',
-    'bootstraps/sport'
+    'bootstraps/sport',
+
+    'enhanced-core'
```

Comment:
i wonder if, with all the name changes, `enhanced-core` is confusing? like a contradiction in terms if you've not been following along closely enough? maybe `enhanced-common`? not a big deal though.